packages/e2e.sandbox.catchall/src/runSubscriptionsTest.ts

```diff
@@ -14,42 +14,83 @@
  * limitations under the License.
  */
 
+import type { ObjectOrInterfaceDefinition } from "@osdk/api";
 import type { EXPERIMENTAL_ObjectSetListener } from "@osdk/api/unstable";
 import { __EXPERIMENTAL__NOT_SUPPORTED_YET_subscribe } from "@osdk/api/unstable";
-import { Employee } from "@osdk/e2e.generated.catchall";
-import { client } from "./client.js";
-import { logger } from "./logger.js";
+import { $Actions, MtaBus, OsdkTestObject } from "@osdk/e2e.generated.catchall";
+import { client, dsClient } from "./client.js";
 
-export function runSubscriptionsTest() {
-  const makeObjectSetListener = (
-    prefix: string,
-  ): EXPERIMENTAL_ObjectSetListener<any> => {
-    return {
+export async function runSubscriptionsTest() {
+  client(OsdkTestObject)[__EXPERIMENTAL__NOT_SUPPORTED_YET_subscribe](
+    [
+      "primaryKey_",
+      "stringProperty",
+    ],
+    {
+      onChange(object) {
+        console.log(
+          "Object with primaryKey ",
+          object.object.primaryKey_,
```

Comment:
this is a real property whose name happens to be `.primaryKey_` right?

Reply:
yes, thats correct